### PR TITLE
Update README to include details about deployment

### PR DESCRIPTION
**Pull request by brenetic**
The deployment process for the app is a little obscure.

https://trello.com/c/EhzSC1vv/366-zendesk-migration-configure-rp-form

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,8 +46,73 @@ be assigned. |
...
+The test is run within Docker. This allows us to create an environment that includes the dependencies required by Selenium to run on our Jenkins setup. The [Jenkins job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-smoke-test/) that runs the smoke test is triggered every 5 minutes and after this application is deployed.
 
 The smoke test has been filtered out of the main test run (`bundle exec rspec spec`) via a `smoke` filter.
 
 To run the smoke test, use: `./smoke-test.sh`.
+
+
+## Deploying
+
+The app is deployed to the PaaS and is called `verify-environment-access`. The production space is called `docs`. There are two other spaces, `onboarding-forms` and `sandbox`, but the app is not currently deployed to those.
```
Comment (phantas) [inline README.md]: it is on onboarding-forms as `verify-environment-access-staging`. There are any other spaces under Verify, I just only gave you access to those (stubs is deployed on PaaS, for example)

File: README.md [hunk 2]
```diff
@@ -46,8 +46,73 @@ be assigned. |
...
+cf set-env verify-environment-access <ENV_VAR> <VALUE>
+```
+
+To see the current environment config for the app:
+
+```
+cf env verify-environment-access
+```
+
+Once a PR is merged to master this triggers a Jenkins [build job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-build/) and then a [deploy job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-deploy/).
```
Comment (phantas) [inline README.md]: Add a paragraph on how to restart the app if we are just changing variables and no PR is being merged.

File: README.md [hunk 3]
```diff
@@ -46,8 +46,73 @@ be assigned. |
...
+cf logs verify-environment-access
+```
+
+The above will give you a real time tail of the logs.
+
+```
+cf logs verify-environment-access --recent
+```
+
+The above will out put all the current log lines in the [Loggregator buffer](https://docs.cloudfoundry.org/loggregator/architecture.html) associated with the app.
```
Comment (galund) [inline README.md]: typo "output"

File: README.md [hunk 4]
```diff
@@ -46,8 +46,73 @@ be assigned. |
...
+To see the current environment config for the app:
+
+```
+cf env verify-environment-access
+```
+
+Once a PR is merged to master this triggers a Jenkins [build job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-build/) and then a [deploy job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-deploy/).
+
+
+## Debugging
```
Comment (phantas) [inline README.md]: ❤️ this section so much!!!!

File: README.md [hunk 5]
```diff
@@ -46,8 +46,73 @@ be assigned. |
...
+The logging output from the Zendesk client is not very verbose. You can add the following to the Zendesk client config in `zendesk_client.rb`:
+
+```
+require 'logger'
+config.logger = Logger.new(STDOUT)
+```
+
+This also logs out unredacted information for the form that is being submitted so do not check this in.
+
+To run the smoke tests locally the you can change the url that the `user_visits_form_smoke_spec` is visiting to `http://host.docker.internal:3000/`. The Docker containers running the smoke tests then should make the requests to your local app.
```
Comment (galund) [inline README.md]: +`.rb`

**Comment by brenetic** [comment]
@phantas @galund Thanks for the comments. Have updated

File: README.md [hunk 6]
```diff
@@ -46,8 +46,83 @@ be assigned. |
...
 To run the smoke test, use: `./smoke-test.sh`.
+
+
+## Deploying
+
+The app is deployed to the PaaS and is called `verify-environment-access`. The production space is called `docs`.
+
+In the `onboarding-forms` space the app is named `verify-environment-access-staging`.
+
+It is not currently deployed to the `sandbox` space.
```
Comment (phantas) [inline README.md]: please remove this line, it introduces confusion. The sandbox space has nothing to do with this, it's just a place for experimentation.

**Pull request merged**